components/org.wso2.carbon.identity.media.endpoint/src/gen/java/org/wso2/carbon/identity/media/endpoint/ContentApi.java

```diff
@@ -0,0 +1,61 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.wso2.carbon.identity.media.endpoint;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.io.File;
+
+import javax.validation.Valid;
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import io.swagger.annotations.*;
+
+@Path("/content")
+@Api(description = "The content API")
+
+public class ContentApi  {
```

Comment:
shouldn't this be MediaAPI?

Reply:
since we discussed to change the url here: https://github.com/wso2/identity-media/pull/14/files/5b5f3e8dd45290d6c77d0f0d5bac9fdc4abf4afd#diff-b90f91dee8a28a753d932f644b3b9ba2R14
this would be the result of the auto-generation of code.